lib/HTTP/Tiny/NoProxy.pm

```diff
@@ -13,7 +13,7 @@ use base 'HTTP::Tiny';
 sub new {
     my ($self, %args) = @_;
 
-    $args{no_proxy} = "127.0.0.1";
+    $args{no_proxy} = [127.0.0.1, 127.0.0.11];
```

Comment:
FYI all of 127/8 is on the local loopback.

Reply:
It is indeed, but we only use 127.0.0.11 and 127.0.0.1, and there's no way to provide a CIDR range / netmask / regex to HTTP::Tiny, it just gets matched as a plain string against the end of `$host` - https://github.com/chansen/p5-http-tiny/blob/master/lib/HTTP/Tiny.pm#L698